tutorials/2d/2d_sprite_animation.rst

```diff
@@ -192,12 +218,37 @@ released.
 
     extends KinematicBody2D
 
+    onready var _animation_player = $AnimationPlayer
+
     func _process(delta):
         if Input.is_action_pressed("ui_right"):
-            $AnimationPlayer.play("walk")
+            _animation_player.play("walk")
         else:
-            $AnimationPlayer.stop()
-
+            _animation_player.stop()
+
+ .. code-tab:: csharp
+
+    class Character : KinematicBody2D
```

Comment:
As per the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html#other-considerations), use explicit access modifiers.

```suggestion
    public class Character : KinematicBody2D
```